frb/halos.py

```diff
@@ -882,6 +882,40 @@ def rho_b(self, xyz):
         #
         return rho
 
+class MB15(ModifiedNFW):
+    """
+    Encodes the Galactic halo profile from
+    Miller & Bregman 2015, ApJ, 800, 14
+    https://ui.adsabs.harvard.edu/abs/2015ApJ...800...14M/abstract
+
+
+    """
+    def __init__(self, log_Mhalo=12.18, c=7.67, f_hot=0.75, **kwargs):
+        # Init ModifiedNFW
+        ModifiedNFW.__init__(self, log_Mhalo=log_Mhalo, c=c, f_hot=f_hot, **kwargs)
+
+        # Best parameters
+        self.beta = 0.45
+        self.n0_rc3b = 0.79e-2  # Last entry of Table 2; Crazy units
+
+    def nH(self, xyz):
+        """
+        Calculate the number density of Hydrogen
+
+        Args:
+            xyz: ndarray
+              Coordinates in kpc
+
+        Returns:
+            ndarray: Number density with units of 1/cm**3
+
+        """
+        radius = np.sqrt(rad3d2(xyz))
+        #
+        #nH = self.n0 * (1+ (radius.self.r_c)**2)**(-3*self.beta/2)
+        nH = self.n0_rc3b / radius**(3*self.beta)
```

Comment:
how about adding a comment say this is from Eq (2) of their paper?

Reply:
and, where is nHe in this equation? should nH be some factor smaller?

Reply:
done.

n_e is a method in the parent class:  `ModifiedNFW`